ui/packages/components/src/RunDetailsV2/RunDetails.tsx

```diff
@@ -57,13 +56,8 @@ export function RunDetails(props: Props) {
     enabled: Boolean(outputID),
```

Comment:
**issue**: It still looks like this query will not execute until the `outputID` is fetched so this won't run in parallel. I think this needs to change to `runID` or remove the enabled config option all together as everything else is passed by props.

Reply:
Oops, good call